blocks/header/header.js

```diff
@@ -1,94 +1,367 @@
+      headerWrapper.classList.toggle('expanded');
+      this.classList.toggle('change');
+      optionsWrapper.classList.toggle('show');
+      hasToggled = true;
+    } else if (window.innerWidth <= 1000) {
+      hasToggled = false;
+    }
+  });
+
+  const optionsWrapper = document.querySelector('.options-wrapper');
+  setTimeout(() => {
+    optionsWrapper.classList.toggle('show');
+  }, 100);
+
+  // Select the first child of mega-menu and all div children of other-options
+  const megaMenuFirstChild = document.querySelector('.mega-menu').firstElementChild;
+
+  const otherOptionsChildren = Array.from(document.querySelector('.other-options').children);
+  const navDivs = [megaMenuFirstChild].concat(otherOptionsChildren);
+  let menuOptions = [];
+
+  // Iterate over each div
+  for (let i = 0; i < navDivs.length; i++) {
+    let menuOption = {};
+    let div = navDivs[i];
+
+    // Find the h2 within this div and assign its innerHTML to the title of menuOption
+    let h2 = div.querySelector('h2');
+    if (h2) {
+      menuOption.title = h2.innerHTML;
+
+      // Find all a tags within this div
+      let links = div.querySelectorAll('a');
+      if (links.length > 0) {
+        menuOption.subMenu = [];
+
+        links.forEach((link) => {
+          let submenuItem = {};
+          // Clone the link element
+          let cloneLink = link.cloneNode(true);
+          submenuItem.name = cloneLink.textContent;
+          submenuItem.url = cloneLink.href;
+
+          // createTags([cloneLink]); // Tags need to be implemented later
+
+          submenuItem.updatedLinkHTML = cloneLink.outerHTML;
+
+          menuOption.subMenu.push(submenuItem);
+        });
+      }
+      menuOptions.push(menuOption);
+    } else {
+      // If there is no h2 in the div, remove the div from the DOM
+      div.parentNode.removeChild(div);
+    }
+  }
+
+  let originalMenuHTML;
+
+  function generateSubMenu(option) {
+    return `<div class='sub-menu-title' data-option='${option.title}'>${option.title}</div>${option.subMenu.map((subMenuItem) => `<a href='${subMenuItem.url}'>${subMenuItem.updatedLinkHTML}</a>`).join('')}`;
+  }
+
+  const menuOptionsHTML = menuOptions.map((option) => `<div class='menu-option' data-option='${option.title}'>${option.title}</div>`).join('');
+
+  originalMenuHTML = menuOptionsHTML;
+
+  optionsWrapper.innerHTML = menuOptionsHTML;
+
+  function handleMenuOptionClick() {
+    const optionTitle = this.dataset.option;
+    const selectedOption = menuOptions.find((opt) => opt.title === optionTitle);
+    optionsWrapper.innerHTML = generateSubMenu(selectedOption);
+
+    const subMenuTitle = document.querySelector('.sub-menu-title');
+    if (subMenuTitle) {
+      subMenuTitle.addEventListener('click', handleSubMenuTitleClick);
+    }
+    const optionWrapperShow = document.querySelector('.options-wrapper.show');
+    optionWrapperShow.style.height = '100vh';
```

Comment:
This line caused a JS error when I reduced the window from "desktop" to "mobile" size.

![Screenshot 2023-07-18 at 18 27 35](https://github.com/hlxsites/bitdefender/assets/143527/0adc0f9c-3790-42e6-9513-cd657afcce12)